gtk4/sample/examples/column_view_demo.rb

```diff
@@ -0,0 +1,76 @@
+# Standalone demo of ColumnView in gtk4.
```

Comment:
Could you add a license header like other file?

You write this script from scratch, right?

Reply:
Its based on a python example I found online.

Reply:
Could you share the URL of the example?
We need to confirm the license of the example.